### PR TITLE
[WebView iOS] add allowsInlineMediaPlayback prop to play inline html5 video

### PR DESCRIPTION
**Pull request by Crash--**
Allow an html5 video to be played inline. (see #3112)

### CODE REVIEW

File: Libraries/Components/WebView/WebView.ios.js [hunk 1]
```diff
@@ -107,6 +107,13 @@ var WebView = React.createClass({
      * user can change the scale
      */
     scalesPageToFit: PropTypes.bool,
+    /**
+    * Determines whether HTML5 videos play inline or use the native full-screen controller.
+    * default value `false`
+    * **NOTE** : "In order for video to play inline, not only does this property need to be set to true, but the video element in the HTML document must also include the webkit-playsinline attribute."
```
Comment (ide) [inline Libraries/Components/WebView/WebView.ios.js]: Please wrap to 80 cols

File: React/Views/RCTWebViewManager.m [hunk 2]
```diff
@@ -34,6 +34,8 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(onLoadingStart, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onLoadingFinish, RCTDirectEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(onLoadingError, RCTDirectEventBlock);
+RCT_REMAP_VIEW_PROPERTY(allowsInlineMediaPlayback,_webView.allowsInlineMediaPlayback,BOOL);
```
Comment (ide) [inline React/Views/RCTWebViewManager.m]: Space after each comma

**Comment by Crash--** [comment]
@ide Done. The [contributing guide](https://github.com/facebook/react-native/blob/master/CONTRIBUTING.md#documentation) says for documentation "do not wrap lines at 80 char"

**Comment by deminoth** [comment]
+1

**Comment by nicklockwood** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**